docs/source/architecture.rst

```diff
@@ -1,17 +1,142 @@
-*****************
-PBFT Architecture
-*****************
+Architecture
+************
+
+The Sawtooth PBFT algorithm is a voting-based consensus algorithm with Byzantine
+fault tolerance, which ensures `safety and liveness
+<https://en.wikipedia.org/wiki/Liveness#Liveness_and_safety>`__.
+The network can tolerate a certain number of "bad" nodes. As long as this number
+is not exceeded, the network will work properly. In addition, blocks committed
+by nodes are final, so there are no forks in the network.
+
+All nodes on the network participate in the consensus process by verifying
+candidate blocks and exchanging consensus messages to communicate their progress
```

Comment:
Requested some changes to wording similar to this in the previous PR; can you make similar changes here? Let me know if you have any questions about specifics.

Reply:
Done